src/ODEmodel.jl

```diff
@@ -44,13 +48,35 @@ function vTOg(v::AbstractVector)
     return G1, G2
 end
 
+""" To find the initial vector. """
+function residue(p, g1, g2)
+    t = LinRange(0.0, 0.5 * size(g1, 1), size(g1, 1))
+    g00 = g1[1, 1] + g2[1, 1]
+    return predict(p, g00, t, g1[:, 1], g2[:, 1])[1]
+end
+
+function optimise_control(g1, g2)
```

Comment:
Are you fitting the control separate from everything else? Shouldn't this be fit with all the other drug conditions?

Reply:
Yes, I didn't include Hill at all, and this was just to find the ODE params of the control for the purpose of finding the initial cell vector. I thought we need that first before fitting other concentrations. If this is not what you intended I can change it.

Reply:
The control should be fit at the same time as everything else. I think you can assume the death rates are 0 for the control.

Reply:
Thanks, I will do this tomorrow.

Reply:
Following up this comment:
We have the death rates =0 for control conditions, and we fit the control along with other concentrations. This step is just to find the control that will then be used to find the initial cells in sub-phases. I mean this will form the pControl argument for `newPredict()`:

![image](https://user-images.githubusercontent.com/34812750/107441169-a69a8200-6ae9-11eb-9d09-b75a136affc2.png)

Please let me know if I am doing it wrong. Thank you.

Reply:
Why does optimize control exist then?

Reply:
As far as I can tell you’re still fitting the control separately. You can’t do this. The best fit for the control will also be influenced by the drug treatment data, due to the Hill relationship.

Reply:
Sorry I couldn't figure out how to do it. 
Could you please clarify a little bit about how to get the control parameters for the g0 vector?
It seems to me that we have to have the control parameters to use it as the g0 vector in `newPredict`. If we run all the conditions, we have to go it with a point initial value, not a vector.

Reply:
p are parameters ultimately calculated from the unknown parameters being fit. pControl is no different. It should be derived from the unknown vector being used in the fitting process for both drugs.

This is exactly how we had it before—the control condition was fit along with the drug effects.

Reply:
The control parameters are the parameters of the model in the absence of drug.

Reply:
I don’t know what you mean about needing an initial point. pControl is passed to startV to calculate the initial point. There’s no contradiction here.

Reply:
Thank you. It should be fixed.